ibm_mq/datadog_checks/ibm_mq/collectors/queue_metric_collector.py

```diff
@@ -75,13 +75,23 @@ def collect_queue_metrics(self, queue_manager):
 
     def discover_queues(self, queue_manager):
         # type: (pymqi.QueueManager) -> Set[str]
+
+        _discover = (
+            self._discover_queues_via_names if self.config.auto_discover_queues_via_names else self._discover_queues
+        )
+
+        if self.config.auto_discover_queues_via_names:
+            self.log.debug("Using _discover_queues_via_names to discover queues")
+        else:
+            self.log.debug("Using _discover_queues to discover queues")
```

Comment:
nit: could we log this in the methods themselves rather than using an extra if statement?

Reply:
Done!